.github/codecov.yml

```diff
@@ -1,16 +1,15 @@
 coverage:
   status:
-    patch:
-      default:
-        target: "80"
     project:
       default:
-        target: auto
-        threshold: 1
+        informational: true
+    patch:
+      default:
+        informational: true
 ignore:
-  - "test/"
+  - "*test*"
   - "benchmark/"
-  - "internal/testutils"
+  - "*testutils*"
```

Comment:
This will be covered by `*test*` right?

Reply:
Also, I'm a little concerned about using wildcards here. How many entries do you have if you were to not use wildcards and instead listed them out explicitly?

Reply:
I actually agree with you. I'll make it explicit since its not that big of a list